engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdown.java

```diff
@@ -137,7 +137,7 @@ public Vector2i getPreferredContentSize(Canvas canvas, Vector2i areaHint) {
     public String getMode() {
         if (!isEnabled()) {
             return DISABLED_MODE;
-        } else if (opened || (TabbingManager.focusedWidget != null && TabbingManager.focusedWidget.equals(this))) {
```

Comment:
This line is important - `ACTIVE_MODE` is what highlights the widget when it is tabbed to. Without checking to see if it's the focused widget (that is, if it's the currently tabbed-to widget), it won't be highlighted when selected by tabbing (unless it's opened). You might be able to find a work-around to avoid using `TabbingManager`, but you still need something to determine if it should be highlighted.